extensions/wikia/ContentReview/tests/ContentReviewHooksTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class ContentReviewHooksTest extends WikiaBaseTest {
+
+	public function setUp() {
+		$this->setupFile = __DIR__ . '/../ContentReview.setup.php';
+		parent::setUp();
+	}
+
+	/**
+	 * In this test case the setSessionData() method should not be called
+	 * because there are no IDs of wikias stored in a session under the test mode key.
+	 */
+	public function testDisableTestModeNoSessionData() {
+		$requestMock = $this->getMockBuilder( 'WebRequest' )
+			->disableOriginalConstructor()
+			->setMethods( [ 'getSessionData', 'setSessionData' ] )
+			->getMock();
```

Comment:
It's simpler to just use `getMock` unless you need to disable the constructor, which you don't here, i.e. just:
`$requestMock = $this->getMock( 'WebRequest', [ 'getSessionData', 'setSessionData' ] );`